bazel/verilog.bzl

```diff
@@ -408,6 +413,10 @@ rule_verilog_fpv_test = rule(
         "tool": attr.string(
             doc = "Formal tool to use. If not provided, default is decided by the BAZEL_VERILOG_RUNNER_TOOL implementation.",
         ),
+        "custom_tcl": attr.label(
+            doc = "custom TCL script to run.",
+            allow_single_file = [".tcl"],
+        ),
```

Comment:
```suggestion
        "appendcustom_tcl": attr.label(
            doc = "Custom TCL script to run after the elaboration step. Do not include Tcl commands that manipulate sources, headers, defines, or parameters, as those will be handled by the rule implementation.",
            allow_single_file = [".tcl"],
        ),
```

Reply:
If not provided, mention that a default set of tcl commands will be run

Reply:
Also need to mention that the tcl script is specific to the tool. We should fail the build if the custom tcl is provided and `tool` is not provided to the rule